src/averageAges.js

```diff
@@ -37,7 +38,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person => person.sex === 'f');
+
+  const mothers = people.map(
+    person => person.mother).filter(mother => mother !== null);
+
+  const isWithChildren = withChildren ? people.filter(
```

Comment:
general convention of naming in java script to use is prefix for boolean variables